app/src/app/components/Sidebar/index.tsx

```diff
@@ -53,16 +74,41 @@ function Sidebar() {
         </div>
         {isUserLoggedIn ? (
           <div className={`column ${styles.contentLinks} start`}>
-            <div className="row m-bottom-3">
+            <div className={`column m-bottom-3 ${styles.selectContainer}`}>
               <span className={styles.techTitle}>Tech:</span>
-              <select className={styles.techSelect} value={selectedTech} onChange={handleTechChange}>
-                <option value="all">Todas</option>
+              <div
+                className={cn(styles.boxTechSelected, {
+                  [`${styles.isOpen}`]: toogleOpen
+                })}
+                onClick={toggleSelectMenu}
+                ref={select}
+              >
+                <span className={styles.optionSelected}>{selectedTech}</span>
+              </div>
+              <ul className={cn(styles.menuSelect, { [`${styles.menuSelectOpen}`]: toogleOpen })}>
+                <li key={ALL_TECHS} className={styles.itemList} onClick={() => handleTechChange(ALL_TECHS)}>
```

Comment:
Hmmm :thinking: I don't like this thing of having boilerplate just to add one option. I think you can do something like send a boolean and add that "all values" props internally. Or maybe send an object with the label of that "all values" option. Something like this:
```js
options = hasAllValues ? [{ label: allValuesLabel, }, ...tech ] : tech

{options.map(option => { ... })
```